packages/core/admin/admin/src/content-manager/pages/EditView/Header/index.js

```diff
@@ -204,6 +206,63 @@ const Header = ({
           />
         </Dialog>
       )}
+      {showPublishConfirmation && (
+        <Dialog
+          onClose={onPublishPromptDismissal}
+          title="Confirmation"
+          labelledBy="confirmation"
+          describedBy="confirm-description"
+          isOpen={showPublishConfirmation}
+        >
+          <DialogBody icon={<ExclamationMarkCircle />}>
+            <Stack spacing={2}>
+              <Flex justifyContent="center" style={{ textAlign: 'center' }}>
+                <Typography id="confirm-description">
+                  {draftCount}
+                  {formatMessage(
+                    {
+                      id: getTrad(`popUpwarning.warning.has-draft-relations.message`),
+                      defaultMessage:
+                        '<b>{count, plural, =0 { of your content relations is} one { of your content relations is} other { of your content relations are}}</b> not published yet.<br></br>It might engender broken links and errors on your project.',
```

Comment:
```suggestion
                        '<b>{count, plural, one { relations is} other { relations are}}</b> not published yet and might lead to unexpected behavior.',
```

I find the current wording quite weird and confusing:

1. the word "engender" is nothing I've see before and I think we can say this easier
2. We don't know if our users display something as a link or whether it will lead to an error, so I'd relax the warning tone a bit.

What do you think @maevalienard @joshuaellis ?

Reply:
yeah i've never heard of engender, you're suggestion works for me 👍🏼

Reply:
agreed, engender is a bit intense. thanks for the suggestion